test/integration/setup.js

```diff
@@ -10,7 +10,12 @@ module.exports = function () {
       name: { type: String },
       code: { type: Number }
     },
-    price: { type: Number }
+    price: { type: Number },
+    postSave: { type: String }
+  })
+
+  ProductSchema.post('save', function () {
+    this.postSave = 'postSave'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/florianholzapfel/express-restify-mongoose/pullRequest?prid=8423230)
```suggestion
    this.postSave = "postSave"
```